light_node/src/snapshot_command.rs

```diff
@@ -432,3 +439,60 @@ fn resolve_block_reference(
             .unwrap_or_else(|| head.block_hash().clone()),
     }
 }
+
+pub fn import_snapshot(from: &Path, tezos_data_dir: &Path) {
+    // remove existing data
+    let to_remove = [
+        tezos_data_dir.join("context"),
+        tezos_data_dir.join("bootstrap_db"),
+    ];
+    fs_extra::remove_items(&to_remove).expect("Failed to remove existing data");
```

Comment:
@adonagy because it can fail in the next step, I think it would be better to just ask the user to remove the files himself and bail-out here.

Reply:
Oh right, good point 👍